en/docs/deploy/secure-userstore-using-pbkdf2.md

```diff
@@ -0,0 +1,82 @@
+# PBKDF2 hashing
+Password-Based Key Derivation Function 2 (PBKDF2) hashing algorithm is a modern hashing algorithm recommended by NIST.  We can use PBKDF2 hashing method to securely store user passwords in user stores. This method reduces the risk of brute-force attacks due to insecure passwords.
+
+This guide walks you through the steps of configuring PBKDF2 as the hashing algorithm of a JDBC userstore.
+
+!!! note
+    Currently, PBKDF2 supports only JDBC userstores of WSO2 Identity Server.
+
+## Configure PBKDF2 hashing
+To configure PBKDF2 hashing on a JDBC user store:
+
+1. On Identity Server management console (`https://<IS_HOST>:<PORT>`) and [create a JDBC user store](../configure-secondary-user-stores).
```

Comment:
Console URL is wrong here, this just has the host and the port.

Reply:
Don't we need to say `Login in to the Identity server....`?

Reply:
```suggestion
1. Login to the Identity Server management console (`https://<IS_HOST>:<PORT>/console`) and [create a JDBC user store](../configure-secondary-user-stores).
```